src/views/docs-view/server.ts

```diff
@@ -47,7 +47,19 @@ import { getCustomLayout } from './utils/get-custom-layout'
  * "latestVersionRef" to the remote content loader config.
  */
 function getBetaLatestVersionRef(slug: string): string | undefined {
-	const hasDevPortalBranch = slug == 'vault' || slug == 'waypoint'
+	const hasDevPortalBranch = [
+		'vault',
+		'waypoint',
```

Comment:
Sneaking this one in here...

```suggestion
		'waypoint',
		'boundary',
```